lib/dream_html.ml

```diff
@@ -865,3 +864,101 @@ module Hx = struct
   let ws_connect fmt = string_attr "data-ws-connect" fmt
   let ws_send = attr "data-ws-send"
 end
+
+module MathML = struct
+  (* Attributes *)
+  let accent = bool_attr "accent"
+  let accentunder = bool_attr "accentunder"
+  let columnspan = int_attr "columnspan"
+  let depth fmt = string_attr "depth" fmt
+
+  let display value =
+    ( "display",
+      match value with
+      | `block -> "block"
+      | `inline -> "inline" )
+
+  let displaystyle = bool_attr "displaystyle"
+  let fence = bool_attr "fence"
+  let framespacing fmt = string_attr "framespacing" fmt
+  let height fmt = string_attr "height" fmt
+  let linethickness fmt = string_attr "linethickness" fmt
+  let lspace fmt = string_attr "lspace" fmt
+  let maxsize fmt = string_attr "maxsize" fmt
+  let minsize fmt = string_attr "minsize" fmt
+  let movablelimit = bool_attr "movablelimit"
+
+  let notation value =
+    ( "notation",
+      match value with
+      | `actuarial -> "actuarial"
+      | `bottom -> "bottom"
+      | `box -> "box"
+      | `circle -> "circle"
+      | `downdiagonalstrike -> "downdiagonalstrike"
+      | `left -> "left"
+      | `longdiv -> "longdiv"
+      | `madruwb -> "madruwb"
+      | `phasorangle -> "phasorangle"
+      | `radical -> "radical"
+      | `right -> "right"
+      | `roundedbox -> "roundedbox"
+      | `top -> "top"
+      | `updiagonalarrow -> "updiagonalarrow"
+      | `updiagonalstrike -> "updiagonalstrike"
+      | `verticalstrike -> "verticalstrike" )
+
+  let rowlines value =
+    ( "rowlines",
+      match value with
+      | `none -> "none"
+      | `solid -> "solid"
+      | `dashed -> "dashed" )
+
+  let rowspan = int_attr "rowspan"
+  let rspace fmt = string_attr "rspace" fmt
+  let scriptlevel fmt = string_attr "scriptlevel" fmt
+  let separator = bool_attr "separator"
+  let stretchy = bool_attr "stretchy"
+  let symmetric = bool_attr "symmetric"
+  let voffset fmt = string_attr "voffset" fmt
+  let xmlns fmt = string_attr "xmlns" fmt
```

Comment:
For valid MathML in an XML document, this can actually only have one value: `xmlns="http://www.w3.org/1998/Math/MathML"`. So we should actually just give it the value so the user doesn't have to type it manually. We are doing the same thing in `SVG.xmlns`: https://github.com/yawaramin/dream-html/blob/f47fbfdc4c713e49f4e9bccb17ec8858c3ff1898/lib/dream_html.ml#L689